<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>512</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>512</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>1024</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>2</width>
    <height>2</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>640</width>
    <height>512</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cantamus</string>
  </property>
  <property name="windowIcon">
   <iconset resource="cantamus.qrc">
    <normaloff>:/images/icons/logo.svg</normaloff>:/images/icons/logo.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>431</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1280</width>
     <height>943</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>640</width>
     <height>512</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>640</width>
     <height>431</height>
    </size>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>19</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>19</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1280</width>
     <height>19</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>2</width>
     <height>1</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>640</width>
     <height>1</height>
    </size>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTemporary">
    <property name="title">
     <string>Temporary</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionShow_help"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTemporary"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1280</width>
     <height>40</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>2</width>
     <height>1</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>640</width>
     <height>40</height>
    </size>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionEditor"/>
   <addaction name="actionVoice_configuration"/>
   <addaction name="actionProject_settings"/>
   <addaction name="actionLipsync_editor"/>
   <addaction name="separator"/>
   <addaction name="actionTo_start"/>
   <addaction name="actionRewind"/>
   <addaction name="separator"/>
   <addaction name="actionPlay"/>
   <addaction name="actionPause"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionFast_forward"/>
   <addaction name="actionTo_end"/>
   <addaction name="separator"/>
   <addaction name="actionSelect"/>
   <addaction name="actionDraw_notes"/>
   <addaction name="actionErase"/>
   <addaction name="separator"/>
   <addaction name="action_voiceIcon"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1280</width>
     <height>22</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>640</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>640</width>
     <height>1</height>
    </size>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionPlay">
   <property name="icon">
    <iconset>
     <normalon>:/images/icons/play.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionEditor">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="cantamus.qrc">
     <normaloff>:/images/icons/button_edit.svg</normaloff>:/images/icons/button_edit.svg</iconset>
   </property>
   <property name="text">
    <string>Editor</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionVoice_configuration">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="cantamus.qrc">
     <normaloff>:/images/icons/soundedit.svg</normaloff>:/images/icons/soundedit.svg</iconset>
   </property>
   <property name="text">
    <string>Voice configuration</string>
   </property>
  </action>
  <action name="actionProject_settings">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="cantamus.qrc">
     <normaloff>:/images/icons/settings.svg</normaloff>:/images/icons/settings.svg</iconset>
   </property>
   <property name="text">
    <string>Project settings</string>
   </property>
  </action>
  <action name="actionLipsync_editor">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="cantamus.qrc">
     <normaloff>:/images/icons/lipsync.svg</normaloff>:/images/icons/lipsync.svg</iconset>
   </property>
   <property name="text">
    <string>Lipsync editor</string>
   </property>
  </action>
  <action name="actionTo_start">
   <property name="icon">
    <iconset resource="cantamus.qrc">
     <normaloff>:/images/icons/tostart.svg</normaloff>:/images/icons/tostart.svg</iconset>
   </property>
   <property name="text">
    <string>To start</string>
   </property>
  </action>
  <action name="actionRewind">
   <property name="icon">
    <iconset resource="cantamus.qrc">
     <normaloff>:/images/icons/rw.svg</normaloff>:/images/icons/rw.svg</iconset>
   </property>
   <property name="text">
    <string>Rewind</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="cantamus.qrc">
     <normaloff>:/images/icons/pause.svg</normaloff>:/images/icons/pause.svg</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="cantamus.qrc">
     <normaloff>:/images/icons/stop.svg</normaloff>:/images/icons/stop.svg</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionFast_forward">
   <property name="icon">
    <iconset resource="cantamus.qrc">
     <normaloff>:/images/icons/ffwd.svg</normaloff>:/images/icons/ffwd.svg</iconset>
   </property>
   <property name="text">
    <string>Fast forward</string>
   </property>
  </action>
  <action name="actionTo_end">
   <property name="icon">
    <iconset resource="cantamus.qrc">
     <normaloff>:/images/icons/toend.svg</normaloff>:/images/icons/toend.svg</iconset>
   </property>
   <property name="text">
    <string>To end</string>
   </property>
  </action>
  <action name="actionDraw_notes">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="cantamus.qrc">
     <normaloff>:/images/icons/draw.svg</normaloff>:/images/icons/draw.svg</iconset>
   </property>
   <property name="text">
    <string>Draw notes</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="cantamus.qrc">
     <normaloff>:/images/icons/hand.svg</normaloff>:/images/icons/hand.svg</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="actionShow_help">
   <property name="text">
    <string>Show help...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionErase">
   <property name="icon">
    <iconset resource="cantamus.qrc">
     <normaloff>:/images/icons/erase.svg</normaloff>:/images/icons/erase.svg</iconset>
   </property>
   <property name="text">
    <string>Erase</string>
   </property>
  </action>
  <action name="action_voiceIcon">
   <property name="icon">
    <iconset resource="cantamus.qrc">
     <normaloff>:/images/icons/singericon.svg</normaloff>:/images/icons/singericon.svg</iconset>
   </property>
   <property name="text">
    <string>Voicebank information</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="cantamus.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showAboutWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShow_help</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showHelpBrowser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionVoice_configuration</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>switchVoiceEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEditor</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>switchNoteEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSelect</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>selectPushed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDraw_notes</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>drawPushed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionProject_settings</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>switchSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionErase</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>deletePushed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_voiceIcon</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showVoiceInfo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPlay</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>playProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPause</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>pauseProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStop</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>stopProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showAboutWindow()</slot>
  <slot>showHelpBrowser()</slot>
  <slot>switchVoiceEdit()</slot>
  <slot>switchNoteEdit()</slot>
  <slot>closeProgram()</slot>
  <slot>selectPushed()</slot>
  <slot>drawPushed()</slot>
  <slot>switchSettings()</slot>
  <slot>deletePushed()</slot>
  <slot>showVoiceInfo()</slot>
  <slot>playProject()</slot>
  <slot>pauseProject()</slot>
  <slot>stopProject()</slot>
  <slot>renderFile()</slot>
  <slot>chooseExp()</slot>
  <slot>noFormant(bool)</slot>
 </slots>
</ui>
